source/includes/pipeline_config/_031-create.md.erb

```diff
@@ -241,6 +243,16 @@ Content-Type: <%= data.apis.versions.pipeline %>; charset=utf-8
 
 Create a pipeline.
 
+<aside class="">
+  <strong>Note:</strong>
+  The create pipeline config API supports an optional header parameter <code>X-pause-pipeline-after-create</code> that
+  allow you to specify whether the pipeline should be paused after create or not. If this header parameter is missing or
+  contains a value other than <i>true</i>, then the created pipeline will not be paused.
```

Comment:
Use `<code/>` not `<i/>`. See https://softwareengineering.stackexchange.com/questions/255366/why-are-the-b-and-i-tags-deprecated.

Also `header parameter` should be called `header field` instead. I'd probably rephrase this as:

> The create pipeline config API supports an optional header field `X-pause-pipeline`. This field, if set to `true`, will create the pipeline in the `paused` state. The additional header field `X-pause-cause` may optionally contain the reason for pausing the pipeline.